components/__snapshots__/alert_banner.test.tsx.snap

```diff
@@ -7,8 +7,9 @@ exports[`Components/AlertBanner should match snapshot 1`] = `
   <div
     className="AlertBanner__icon"
   >
-    <i
-      className="icon-alert-circle-outline"
+    <InformationOutlineIcon
+      color="currentColor"
```

Comment:
Can we omit the "currentColor" usages in this PR? That's the [default value](https://github.com/mattermost/compass-icons/blob/master/generate-data.js#L68) already, and I don't think the default value for that is going to change.